cmds/core/dd/dd.go

```diff
@@ -511,5 +451,5 @@ func main() {
 		log.Fatal(err)
 	}
 
-	progress.end()
+	progress.End()
```

Comment:
would it make sense to call it Close?

Reply:
Hm - this marks the end of the process. I find it "End" fits more than "Close".